src/graphql/schema/Trip/typeDefs.gql

```diff
@@ -42,26 +46,52 @@ type TimeFrameObject {
 }
 
 type TimeFrame {
-  id: ID!
-  startDate: String!
-  endDate: String!
-  voters: [User]!
-  creator: User!
+  startDate: String
+  endDate: String
+  voters: [User]
+  creator: User
 }
 
 type Budget {
-  value: ID!
-  name: String!
-  voters: [User]!
-  creator: User!
+  id: ID!
+  value: Int!
+  voters: [User]
+  creator: User
```

Comment:
same here, creator must be required, because without creator there is no Trip to start

Reply:
Agree, think that was just removed because at the time we did not have creators yet.

- [ ] make voters and creator mandatory for Budget
- [ ] make voters and creator mandatory for TimeFrame
- [ ] make voters and creator mandatory for Destination